src/Secret.php

```diff
@@ -54,12 +57,27 @@ public function getSecretVersions(string $secretName) {
      * @param string|null $secretVersion
      * @return SecretEntity
      */
-    public function getSecret($secret, string $secretVersion = null) {
+    public function getSecret($secret, string $secretVersion = null)
+    {
+        // Set cache location
+        $cache = new ApcuCachePool();
+
         if ($secret instanceof SecretVersionEntity && !$secretVersion) {
             $secretVersion = $secret->id;
             $secret = $secret->name;
         }
 
+        $wordpressSecretVersion = $_ENV['WORDPRESS_SECRET_VERSION'] ?? '';
```

Comment:
Should we add something about this config value to the readme?

Reply:
That's, like, so much work. but fine I've added that bad boy.